docs/user/ppl/functions/json.rst

```diff
@@ -60,3 +60,107 @@ Example::
+
+Return type: STRING
+
+Example::
+
+    > source=json_test | eval test_json = json_object('key', 123.45) | head 1 | fields test_json
+    fetched rows / total rows = 1/1
+    +-------------------------+
+    | test_json               |
+    |-------------------------|
+    | {"key":123.45}          |
+    +-------------------------+
+
+JSON_ARRAY
+----------
+
+Description
+>>>>>>>>>>>
+
+Usage: `json_array(element1, element2, ...)` create a json array string with elements.
+
+Argument type: element1: ANY, element2: ANY ...
+
+Return type: STRING
+
+Example::
+
+    > source=json_test | eval test_json_array = json_array('key', 123.45) | head 1 | fields test_json_array
+    fetched rows / total rows = 1/1
+    +-------------------------+
+    | test_json_array         |
+    |-------------------------|
+    | ["key",123.45]          |
+    +-------------------------+
+
+JSON_ARRAY_LENGTH
+----------
+
+Description
+>>>>>>>>>>>
+
+Usage: `json_array_length(value)` parse the string to json array and return size, if can't be parsed, return null
+
+Argument type: value: STRING
+
+Return type: INTEGER
+
+Example::
+
+    > source=json_test | eval array_length = json_array_length("[1,2,3]") | head 1 | fields array_length
+    fetched rows / total rows = 1/1
+    +-------------------------+
+    | array_length            |
+    |-------------------------|
+    | 3                       |
+    +-------------------------+
+
+    > source=json_test | eval array_length = json_array_length("{\"1\": 2}") | head 1 | fields array_length
+    fetched rows / total rows = 1/1
+    +-------------------------+
+    | array_length            |
+    |-------------------------|
+    | null                    |
+    +-------------------------+
+
+JSON_EXTRACT
+----------
+
+Description
+>>>>>>>>>>>
+
+Usage: `json_extract(json_string, path1, path2, ...)` it first transfer json_string to json, then extract value using paths. If only one path, return the value, otherwise, return the list of values. If one path cannot find value, return null as the result for this path. The path use "{<index>}" to represent index for array, "{}" means "{*}".
+
+Argument type: json_string: STRING, path1: STRING, path2: STRING ...
+
+Return type: STRING
+
+Example::
+
+    > source=json_test | eval extract = json_extract('{\"a\": [{\"b\": 1}, {\"b\": 2}]}', 'a{}.b') | head 1 | fields extract
+    fetched rows / total rows = 1/1
+    +-------------------------+
+    | test_json_array         |
+    |-------------------------|
+    | [1,2]                   |
+    +-------------------------+
+
+     > source=json_test | eval extract = json_extract('{\"a\": [{\"b\": 1}, {\"b\": 2}]}', 'a{}.b', 'a{}'') | head 1 | fields extract
+    fetched rows / total rows = 1/1
+    +---------------------------------+
+    | test_json_array                 |
+    |---------------------------------|
+    | [[1,2],[{\"b\": 1}, {\"b\": 2}]]|
```

Comment:
`[[1,2],[{\"b\": 1}, {\"b\": 2}]]` is not a valid JSON array.

Reply:
I update it. Please check the doc.